src/main/java/frc/robot/PowerController.java

```diff
@@ -0,0 +1,168 @@
+package frc.robot;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import edu.wpi.first.wpilibj.PowerDistribution;
+import frc.robot.subsystems.tank.TankSubsystem;
+import static frc.robot.Constants.TankConstants.*;
+
+public class PowerController {
+
+    private static PowerDistribution PDP = new PowerDistribution();
+
+    private ControllableSubsystem tankSubsystem;
+
+//    private ControllableSubsystem climbSubsystem;
```

Comment:
Alignment

Also, is there a reason you have an empty line between each subsystem? I would think they'd make a nice block of 4 or so line.